docs/index.html

```diff
@@ -260,20 +260,61 @@ <h3>CCG Standards Stack</h3>
+          to indicate that presentations relate to the same type of workflow. 
+        </p>
         <p>
-          Workflow <em>instances</em> indicate that separate individual presentations are part of the very same workflow. The majority
-          of data elements in a "10+2" Importer Security Filing must be presented to CBP no later than 24 hours before vessel
-          loading, and the remaining data elements no later than 24 hours before arrival. Specifying the same workflow instance indicates
-          to CBP that the two presentations are part of the same import workflow.
+          Workflow <em>instances</em> indicate that separate individual presentations are part of the very same workflow.
         </p>
+          
+        <section class="informative">
+          <h3>Workflows for US Customs and Border Protection (US CBP)</h3>
+          <img src="workflow-def-ex.png" alt="workflow-def-ex" width="75%">
+          <p>
+            In the above example, two separate 7501 Entry
+            Summaries are submitted to US CBP at different points in time, and related to different imports. By specifying the same
+            workflow definition, it is made clear to US CBP that both relate to the same types of import workflows.
+          </p>
+          <img src="workflow-inst-ex.png" alt="workflow-inst-ex" width="75%">
+          <p>
+            The majority of data elements in a "10+2" Importer Security Filing must be presented to CBP no later than 24 hours before vessel
+            loading, and the remaining data elements no later than 24 hours before arrival. Specifying the same workflow instance indicates
+            to US CBP that the two presentations are part of the same import workflow.
+          </p>
+          <p>
+            A workflow definition indended for use with US CBP imports MUST have a corresponding 
+            plain text <em>manifest document</em> that may be uniquely identified, referenced, 
+            and retrieved by a third party in a publicly accessible and documented manner by the 
+            UUID of the workflow definition.
+          </p>
+          <p>
+            This workflow manifest document MUST list each required credential from the 
+            <a href="https://w3c-ccg.github.io/traceability-vocab/#terminology">
+              Traceability Vocabulary
+            </a>, one per line, and identified by the "type" of credential,
+            for the workflow to be considered "complete".  Completeness
+            in this sense refers to an optimal state where all possible information that
+            can be provided in relation to that workflow definition is present.
+          </p>
+          <p>
+            Following the credential "type" a space, followed by a numeric value that ranges from
+            0 to 1 MAY be added.  If this value is not provided a value of 1 MUST by inferred by
+            any system using this property.  This numeric value SHALL be referred to as "weight",
```

Comment:
`SHALL` is technically an RFC 2119 synonym of `MUST`, but infrequently enough used that it is often misinterpreted (or mis-meant) as `SHOULD`, so I would change this `SHALL` to `MUST`.
```suggestion
            any system using this property.  This numeric value MUST be referred to as "weight",
```

Reply:
Good catch, thanks